mne/commands/mne_freeview_bem_surfaces.py

```diff
@@ -30,6 +30,13 @@ def freeview_bem_surfaces(subject, subjects_dir, method):
     """
     subjects_dir = get_subjects_dir(subjects_dir, raise_error=True)
 
+    if subject is None:
+        raise ValueError("subject argument is None.")
+    if not op.isdir(op.join(subjects_dir, subject)):
```

Comment:
I would use a variable to store `op.join(subjects_dir, subject)`
so to avoid duplicating the function call.

thanks